core/utils/wasm-builder/src/wasm_project.rs

```diff
@@ -266,7 +266,8 @@ fn build_project(project: &Path) {
 
 	match build_cmd.status().map(|s| s.success()) {
 		Ok(true) => {},
-		_ => panic!("Failed to compile WASM binary"),
+		// Use `process.exit(1)` to have a clean error output.
+		_ => process::exit(1),
```

Comment:
Shall we print something to stderr, like: "WASM Build failed check logs at wasm_build.logs"?

Reply:
The error messages are printed to stdout/stderr and I would not like to pipe it to a file.